### ISSUE TITLE
0.6.4 node.dll entry point not found

### ISSUE THREAD

**Issue by creative-resort** [ramboxapp/community-edition#2126]
### Steps to reproduce

1. Update to version 0.6.4 on Windows 7 x64
2. run Rambox

### Expected behavior

Rambox should launch

### Actual behavior

An Error is reported in a dialog box
Entry point in node.dll not found
![2019-03-15_094145](https://user-images.githubusercontent.com/12367437/54419698-69b08f80-4708-11e9-9723-dad90446a50e.png)

### ENVIRONMENT

> Rambox: v0.6.4

> OS: Win 7 x64

**Comment by oliver-graetz** [comment]
I have a similar problem, but with another entry point. The program doesn't start.

![Rambox exe entry point not found](https://user-images.githubusercontent.com/4057601/54823814-40b07180-4ca9-11e9-9bed-54dabec7a428.png)

**Comment by frommie** [comment]
Hi, I had the same problem. Try to deactivate your antivirus software and reinstall rambox again (just run the setup file, don't uninstall rambox). Solved the issue for me. I'm using Avast Free Antivirus.

**Comment by oliver-graetz** [comment]
Nope. There's just regular Windows Defender that comes with Windows 10 in use here. Deactivating its realtime protection didn't make any difference. Uninstalling 0.6.4 and reinstalling 0.6.3 works without problems and the old version runs just fine.

Also, if the antivirus software leads to such problems after installing software **without telling you it did _anything_** would be an unforgivable offense on the antivirus' side. I would get rid of such software immediately.

**Comment by creative-resort** [comment]
indeed
Thank you @frommie
it was a security software issue. The blocked files were listed and after unblocking it now seems to work.

**Comment by oliver-graetz** [comment]
The issue is not resolved for me. I still get the error.

**Comment by frommie** [comment]
I also think the issue is not resolved. This is only a workaround. With the next release we would likely have the same issue again. 
We could debate over the usefulness of antivirus software but in my opinion we should examine why this issue exists in 0.6.4 but not in 0.6.3.

**Comment by oliver-graetz** [comment]
For me, this is not an Antivirus problem. I disabled Windows Defender, and it still flat out doesn't start. I can't run the new version. Period.

**Comment by oliver-graetz** [comment]
Update: Installing "just for me" works, just the system-wide installation bugs out.

**Comment by fvulich** [comment]
I'm gonna close this ticket because seems to be fixed now, feel free to open another ticket if the problem persists.
